packages/tools/client-debugger/client-debugger-view/src/components/FluidValueView.tsx

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { FluidObjectValueNode } from "@fluid-tools/client-debugger";
+import { Accordion } from "./utility-components";
+
+/**
+ * {@link ValueView} input props.
+ */
+export interface FluidValueViewProps {
+	node: FluidObjectValueNode;
+}
+
+/**
+ * Render data with type {@link VisualNodeKind.FluidValueNode}.
+ */
+export function FluidValueView(props: FluidValueViewProps): React.ReactElement {
+	const { node } = props;
+
+	return (
+		<Accordion
```

Comment:
I don't think we need to use an Accordion for this, since there aren't real children. We have our node's value to display, but we're already displaying that in the header.